admin/client/src/components/ListGroup/LIstGroup.scss

```diff
@@ -0,0 +1,96 @@
+// List-group, based on Bootstraps list-group
+// TODO split out campaign specific css from generic list-group
+
+.list-group {
```

Comment:
I made the other amendments but the file name seems like it hasn't updated, I take it this is something you can do @tractorcow ?